usr/src/cmd/zfs/zfs_main.c

```diff
@@ -5158,6 +5173,9 @@ print_uge_deleg_perms(uu_avl_t *who_avl, boolean_t local, boolean_t descend,
 				case ZFS_DELEG_EVERYONE:
 					who = gettext("everyone");
 					who_name = NULL;
+
+				default:
+					break;
```

Comment:
we are outside of switch here without 'default' too.
in this case - we are updating 'who' based on 'who_type'
ZFS_DELEG_CREATE* & ZFS_DELEG_NAMED* no need update here.